include/swift/AST/Decl.h

```diff
@@ -6765,6 +6766,9 @@ inline EnumElementDecl *EnumDecl::getUniqueElement(bool hasValue) const {
 std::pair<DefaultArgumentKind, Type>
 getDefaultArgumentInfo(ValueDecl *source, unsigned Index);
 
+/// Display ValueDecl subclasses.
+void simple_display(llvm::raw_ostream &out, const ValueDecl *&decl);
```

Comment:
Why the reference here?

Reply:
Hmmm. The reference is pointless. I was directly specializing the more-generic version of this function. Will fix when I can tolerate touching `Decl.h`

Reply:
(It's fixed now)